templates/get-involved.twig

```diff
@@ -36,10 +36,10 @@
 				</div>
 				<div class="panel-body">
 					<p>{% trans %}You can help us by improving this website or documenting the software. The website sources are in our
-					   GitHub repo <a href="//github.com/LMMS/lmms.io">LMMS/lmms.io</a>, our documentation is hosted using a
-					   <a href="//lmms.io/wiki">MediaWiki instance</a>.{% endtrans %}</p>
+					   GitHub repo <a href="//github.com/LMMS/lmms.io">LMMS/lmms.io</a>, our documentation is hosted on
+					   <a href="//docs.lmms.io/">GitBook</a>.{% endtrans %}</p>
 					<p>{% trans %}Public registration on our Wiki is disabled due to spam issues. Please post to the
```

Comment:
You might consider omitting this paragraph if the wiki registration spam issues are talking about the wiki that's been shut down. I'm not sure if your current documentation has the same issues, but I'm assuming that this is talking specifically about the MediaWiki.

Reply:
I suppose the spam part isn't relevant, but access to edit the docs is still something people need to request access to. Will edit that line before this merges.